src/module.c

```diff
@@ -9512,6 +9531,20 @@ void moduleInitModulesSystem(void) {
     pthread_mutex_lock(&moduleGIL);
 }
 
+void modulesCron(void) {
+    /* For two reasons, we limit max client count to be freed here. First, if
+     * there are many clients to be freed, this loop might become slow.
+     * Second, we are giving temp clients more chance to be reused. If they are
+     * not used, they'll be freed eventually rather than all at once. */
```

Comment:
you have a comment specifying the throttling, but i feel you're missing a line that specifies what that does (puts the throttling comment in some context)